src/clients/common/notificationDialog.js

```diff
@@ -0,0 +1,329 @@
+    clearNotificationDialogs();
+
+
+  //************************************************************************************** */
+  // Helper functions for createNotificationDialog:
+
+  function getNewNotificationDialog(html) {
+    const notificationDialogDocument = getNotificationDialogDocument();
+    const notificationDialogTemplate = notificationDialogDocument.getElementById('notificationDialogTemplate');
+    const notificationDialogFragment = notificationDialogTemplate.content.cloneNode(true);
+    const notificationDialogElement = notificationDialogFragment.getElementById('notificationDialog');
+    return notificationDialogElement;
+
+    function getNotificationDialogDocument() {
+      const parser = new DOMParser();
+      const notificationDialogDocument = parser.parseFromString(html, 'text/html');
+      return notificationDialogDocument;
+    }
+
+  }
+
+  function checkNotificationDialogType() {
+
+    if (typeof notificationDialogContent === 'string' || notificationDialogContent instanceof String) {
+      return 'string';
+    }
+
+    if (typeof notificationDialogContent === 'object' && Object.keys(notificationDialogContent).length !== 0 && Object.getPrototypeOf(notificationDialogContent) === Object.prototype) {
+      return 'object';
+    }
+  }
+
+  function createDefaultNotificationDialog(notificationDialogString) {
+    createCustomNotificationDialog({text: notificationDialogString});
+  }
+
+  function createCustomNotificationDialog(notificationDialogObject) {
+    const {style, text, linkButton, title} = notificationDialogObject;
+
+    setNotificationDialogStyle();
+    addTextToNotificationDialog(text);
+    addLinkButton();
+
+    //if title is provided use that but if not use default
+    if(title){notificationDialogElement.querySelector(`.notificationdialog-title`).innerHTML = titleText;}
+    else{
+      addDefaultStyleBasedTitle();
+    }
+    //close button can be optional
+    if(canDismiss){
+      notificationDialogElement.querySelector(`.notificationdialog-close`).style.visibility = 'hidden';
+    }
+    else{
+      addCloseButton(notificationDialogObject);
+    }
+
+    function setNotificationDialogStyle() {
+
+      // style 'info' is the default status message
+      // notificationDialog has blue background with info icon
+      let backgroundColor = 'var(--color-blue-60)';
+      let iconColor = 'var(--color-blue-100)';
+      let icon = 'error_outline';
+
+
+      if (style === 'success') {
+        backgroundColor = 'var(--color-green-80)';
+        iconColor = 'var(--color-green-100)';
+        icon = 'check_circle_outline';
+      }
+
+      if (style === 'alert') {
+        backgroundColor = 'var(--color-yellow-80)';
+        iconColor = 'var(--color-blue-100)';
+        icon = 'warning_amber';
+      }
+
+      if (style === 'error') {
+        backgroundColor = 'var(--color-red-80)';
+        iconColor = 'var(--color-red-100)';
+        icon = 'report_gmailerrorred';
+      }
```

Comment:
use style helper function mentioned in `src/clients/common/notificationBanner.js`